disconnect-blacklist.json

```diff
@@ -7399,6 +7503,13 @@
           ]
         }
       },
+      {
+        "Yandex": {
+          "http://www.yandex.com/": [
+            "api-maps.yandex.ru"
```

Comment:
Does this need a corresponding `Yandex.resources` entry under `disconnect-entitylist.json` ?